backends-common/elasticsearch/pom.xml

```diff
@@ -105,4 +105,17 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <reuseForks>true</reuseForks>
+                    <forkCount>2</forkCount>
```

Comment:
It's at the same time too high and too low: if we can do parallelism, why stop at 2? We should use 0.5C or something like that

Reply:
So far we only start 2 forks for Cassandra related projects to not obliterate the node RAM, I followed the same approach for ElasticSearch.

There is a balance between sstarting too much ES nodes and running code concurrently. A parralelism of 2 yield better results locally.

Reply:
With a specific environment, probably. But maven configuration is not supposed to target a specific environment.
I suggest we remove these tweaks and add them as environment variables on the CI.
We will then tweak value for each host.